examples/batch-mpi.yaml

```diff
@@ -0,0 +1,151 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+blueprint_name: batch-wrf-mpi
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: batch-wrf
+  region: us-central1
+  zone: us-central1-c
+
+deployment_groups:
+- group: primary
+  modules:
+  - id: network1
+    source: modules/network/pre-existing-vpc
+
+  ### Software & Script Installation ###
+  - id: sharefs
+    source: modules/file-system/filestore
+    use: [network1]
+    settings:
+      local_mount: /share
+
+  - id: spack
+    source: community/modules/scripts/spack-install
+    settings:
+      install_dir: /share/spack
+      configs:
+      - type: file
+        scope: defaults
+        content: |
+          config:
+            build_stage:
+              - /share/spack/spack-stage
+      - type: file
+        scope: defaults
+        content: |
+          modules:
+            default:
+              tcl:
+                hash_length: 0
+                all:
+                  conflict:
+                    - '{name}'
+                projections:
+                  all: '{name}/{version}-{compiler.name}-{compiler.version}'
+      compilers:
+      - gcc@8.2.0 target=x86_64
```

Comment:
is using an out-of-date gcc intentional or just a matter of working from existing examples?

FYI: gcc 10.3 is first release to support optimizations for c2d.

Updating this can be a later task, I'm mostly curious.

Reply:
The cache we use for testing was built with this gcc. I will open a task for this.

Reply:
Just for reference, this is actually required. Intel-MPI 2018 is not compatible any GCC newer than series 8 for Fortran applications.